packages/openapi-typescript/test/schema-object.test.ts

```diff
@@ -374,6 +374,11 @@ describe("Schema Object", () => {
       const generated = transformSchemaObject({}, options);
       expect(generated).toBe(`unknown`);
     });
+
+    test("unknown nullable", () => {
+      const generated = transformSchemaObject({ nullable: true }, options);
+      expect(generated).toBe(`unknown`);
+    });
```

Comment:
The issue suggested the type should be `unknown | null`, but `unknown` is effectively the same and `tsUnionOf()` is won't do a union if the type is already `unknown`.